pages/sdk/protocol-kit/guides/safe-deployment/deploy-safe.mdx

```diff
@@ -0,0 +1,139 @@
+import { Steps } from 'nextra/components'
+
+# Deploy a Safe
+
+In this guide, you will learn how to deploy a new Safe using the Protocol Kit. This process includes initializing the Protocol Kit, setting up your Safe configuration, and executing the deployment.
+
+For more detailed information, see the [Protocol Kit Reference](../../../protocol-kit/reference/safe.mdx).
+
+## Prerequisites
+
+- [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+## Install dependencies
+
+First, you need to install the Protocol Kit.
+
+```bash
+pnpm add @safe-global/protocol-kit
+```
+
+## Steps
+
+<Steps>
+
+  ### Imports
+
+  Begin by importing the `Safe` class from the Protocol Kit.
+
+  {/* <!-- vale off --> */}
+
+  ```typescript
+  import Safe from '@safe-global/protocol-kit'
+  ```
+
+  {/* <!-- vale on --> */}
+
+  ### Create a signer
+
+  Firstly, you need to get a signer, this example uses a private key, but any way to get an `EIP-1193` compatible signer can be used.
```

Comment:
This is not super accurate. Something like this could be better? :

"You need a signer to instantiate the protocol-kit. In the following example a private key is being used but you can specify as well an address. More about the available options in the [protocol-kit reference](https://safe_deployment_guide_v2--docs.review.5afe.dev/sdk/protocol-kit/reference/safe#init)"

Reply:
done!